nixos/modules/services/misc/mautrix-telegram.nix

```diff
@@ -93,12 +93,19 @@ in {
         default = null;
         description = ''
           File containing environment variables to be passed to the mautrix-telegram service,
-          in which secret tokens can be specified securely by defining values for
+          in which secret tokens can be specified securely by defining values for e.g.
           <literal>MAUTRIX_TELEGRAM_APPSERVICE_AS_TOKEN</literal>,
           <literal>MAUTRIX_TELEGRAM_APPSERVICE_HS_TOKEN</literal>,
           <literal>MAUTRIX_TELEGRAM_TELEGRAM_API_ID</literal>,
           <literal>MAUTRIX_TELEGRAM_TELEGRAM_API_HASH</literal> and optionally
           <literal>MAUTRIX_TELEGRAM_TELEGRAM_BOT_TOKEN</literal>.
+          </para>
+
+          <para>
```

Comment:
Blindly relying on how the manual is assembled (i.e. that each description is inside a `<para>` *right now*) is IMHO not a good idea, this can easily cause some weird side-effects as soon as some config of our docbook build changes.